apps/test-low-memory.sh

```diff
@@ -48,7 +48,7 @@ ${PARALLEL} <<SCRIPT
 npm run lint
 (PORT=9876 ${GRUNT_CMD} unitTest && ${CODECOV} -cF unit) ${LOG} log/unitTest.log
 (PORT=9877 $GRUNT_CMD storybookTest && ${CODECOV} -cF storybook) ${LOG} log/storybookTest.log
-(PORT=9878 $GRUNT_CMD scratchTest && ${CODECOV} -cF scratch) ${LOG} log/scratchTest.log
+# (PORT=9878 $GRUNT_CMD scratchTest && ${CODECOV} -cF scratch) ${LOG} log/scratchTest.log
```

Comment:
is this intentional?

Reply:
yes but it goes along with disabling the scratch test - I will comment